src/actions/actionTypes.js

```diff
@@ -7,10 +7,12 @@ export const SIGN_IN_SUCCESS = 'SIGN_IN_SUCCESS';
 export const CREATE_TARGET = 'CREATE_TARGET';
 export const LOAD_TARGETS = 'LOAD_TARGETS';
 export const UPDATE_TARGET = 'UPDATE_TARGET';
-export const UPDATE_FREE_TARGET = 'UPDATE_FREE_TARGET';
 export const DELETE_TARGET = 'DELETE_TARGET';
 export const RESET_TARGETS = 'RESET_TARGETS';
+export const UPDATE_FREE_TARGET = 'UPDATE_FREE_TARGET';
 export const RESET_FREE_TARGET = 'RESET_FREE_TARGET';
+export const UPDATE_SELECTED_TARGET = 'UPDATE_SELECTED_TARGET';
```

Comment:
Yeah, I agree with @bernabe9. It might be nice to have, but it's just a detail and seems unnecessary to me.